.travis.yml

```diff
@@ -21,7 +21,7 @@ matrix:
         - php: 7.1
           env: TASK_SCA=1
         - php: nightly
-          env: TASK_SCA=1 COMPOSER_FLAGS="--ignore-platform-reqs"
+          env: COMPOSER_FLAGS="--ignore-platform-reqs" TASK_CS=0
```

Comment:
what I was thinking actually was introduction of new ENV flag:
`PHP_CS_FIXER_IGNORE_ENV=1` and **allow** tool to run on 7.2 with that flag.
For that, we would be able to run the SCA